tests/integration/local/start_api/test_start_api_cdk_template.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+
+import pytest
+import requests
+from parameterized import parameterized_class
+
+from tests.integration.local.start_api.start_api_integ_base import StartApiIntegBaseClass
+
+
+@parameterized_class(
+    ("template",),
```

Comment:
Comparing with `test_start_api.py`, should this be `tempalte_class` instead?

Reply:
You're right, it should be `template_path` instead.